app/presenters/document_export_presenter.rb

```diff
@@ -68,6 +68,18 @@ def present_attachments(edition)
                          end
       attachment.as_json(include: :attachment_data, methods: %i[url type])
                 .merge(govspeak_content)
+                .merge(variants: attachment_variants(attachment))
+    end
+  end
+
+  def attachment_variants(attachment)
+    return [] unless attachment.try(:attachment_data)
```

Comment:
Should this return a hash to be consistent with the next statement?

Reply:
Whoops - good spot - it should!